citar.el

```diff
@@ -239,8 +239,9 @@ If nil, single resources will open without prompting."
   :group 'citar
   :type '(function))
 
-(defcustom citar-keys-with-notes-functions
-  '(citar-file--keys-with-file-notes)
+(defcustom citar-note-backends
```

Comment:
I think I prefer your name for this, `citar-has-note-functions`. It immediately conveys that it's a list of functions, and that it's closely related to the `citar-has-note` function.